modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -367,15 +367,15 @@
 
 // Donation reward for Random516
 /obj/item/clothing/gloves/fingerless/blutigen_wraps
-	name = "Blutigen wraps"
-	desc = "The one who wears these had everything and yet lost it all..."
+	name = "Bandage like wraps"
```

Comment:
why is it "bandage-like" just call it a bandage

Reply:
because bandage wraps are likely to exist and having two things with the same name sucks.